content/docs/beyond-the-basics/verified-domain/index.md

```diff
@@ -0,0 +1,59 @@
+---
+ title: "Verified Domain"
+ description: "Domain Verification and Why It's Important"
+ lead: "Domain Verification and Why It's Important"
+ date: 2021-05-18T14:52:25+01:00
+ lastmod: 2021-05-18T14:52:25+01:00
+ draft: false
+ images: []
+ menu:
+   docs:
+     parent: "beyond-the-basics"
+ weight: 43
+ toc: true
+---
+
+## What is domain verification?
+
+Domain verification assures that actors claiming to be part of an organization are authorized to share information on their behalf. If an organization's tenancy has been verified by the RKVST team, a badge indicating that they have been verified will appear next to their domain name.
+
+##include image of verified domain badge##
+
+{{< note >}}
+**Note:** Having a verified domain is different from a `Tenant Display Name`. Tenant display names are internal, appearing only within your own tenancy, and are not visible to anyone you share with. A verified domain name must be set by the RKVST team, and will be visible to actors outside your tenancy. 
```

Comment:
This note is in response to the ticket for friendly names for RKVST tenancies, to distinguish between the Tenant Display Name and verified domain name.